js/widget.js

```diff
@@ -38,6 +39,15 @@ const ModMap = {
   default: `sgv__last--`,
 };
 
+const updateAlwaysOnTopButton = (isAlwaysOnTop) => 
+  Buttons.alwaysOnTop.querySelector(`img`).src = isAlwaysOnTop ? `asset/icons8-pin-filled-25.png`:`asset/icons8-pin-hollow-25.png`;
+updateAlwaysOnTopButton(await window.electronAPI.isAlwaysOnTop());
+
+Buttons.alwaysOnTop.addEventListener(`click`, async() => {
```

Comment:
This duplicates the functionality and overloads the UI. It should be managed either from the main window or the settings.